src/containers/BooksList.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import Book from '../components/Book';
+
+function BooksList() {
+  const books = useSelector((state) => state.booksReducer);
```

Comment:
As specified by the projects requirement  it uses `mapStateToProps` to fetch books from the `store`

Reply:
Hi there, Taiwo in the requirements for this project states that I can either fetch the books from the state by using connect or redux hooks I chose the latter.